slices/python3-distutils.yaml

```diff
@@ -0,0 +1,99 @@
+package: python3-distutils
+
+essential:
+  - python3-distutils_copyright
+
+slices:
+  python3-11:
+    contents:
+      # Due to conflicts with libpython3.11-stdlib_distribution and chisel not able to reconcile
+      # 2 slices doing glob on the same folder, I have to specify the files explicitely
+      # Also the original package uses hardlink for the common files between python 3.10 and
+      # python3.11 chisel doesn't support hardlinks so I need to use copy
```

Comment:
```suggestion
      # Due to conflicts with libpython3.11-stdlib_distribution and chisel not being able to
      # reconcile 2 slices with globs on the same folder, I have to specify the files explicitly.
      # Also the original package uses hardlink for the common files between python3.10 and
      # python3.11. Chisel doesn't seem to support hardlinks so I need to use copy as a workaround
```

Reply:
@letFunny can you please confirm this "hardlinks" issue. And if it really is a bug, can you please open an issue for it?

Reply:
Forgot to reply here that it is in fact a bug and we opened an issue for it.